src/client/scripts/esm/util/math.js

```diff
@@ -472,6 +472,16 @@ function roundUpToPowerOf2(num) {
 	return Math.pow(2, Math.ceil(Math.log2(num)));
 }
 
+/**
+ * 
+ * @param {Number} s 
+ * @param {Number} e 
+ * @param {Number} progress 
+ */
+function moveTowards(s, e, progress) {
```

Comment:
Could you add a description for the method and each of the arguments? It's not clear what this does.